### PR TITLE
add a build target for CentOS Stream 9

### PR DESCRIPTION
**Pull request by duritong**
Requires https://github.com/lest/centos-rpm-builder-docker/pull/5

### CODE REVIEW

**Comment by karlism** [comment]
```
 docker run --rm \
	-v /home/runner/work/prometheus-rpm/prometheus-rpm/alertmanager:/rpmbuild/SOURCES \
	-v /home/runner/work/prometheus-rpm/prometheus-rpm/_dist9:/rpmbuild/RPMS/x86_64 \
	-v /home/runner/work/prometheus-rpm/prometheus-rpm/_dist9:/rpmbuild/RPMS/noarch \
	-v /home/runner/work/prometheus-rpm/prometheus-rpm/_cache_dnf:/var/cache/dnf \
	ghcr.io/lest/centos-rpm-builder:stream9 \
	build-spec SOURCES/autogen_alertmanager.spec
Unable to find image 'ghcr.io/lest/centos-rpm-builder:stream9' locally
docker: Error response from daemon: manifest unknown.
See 'docker run --help'.
make: *** [Makefile:145: build9-alertmanager] Error 125
Error: Process completed with exit code 2.
```

@lest, we will need CentOS/RHEL images in order to support this

**Comment by duritong** [comment]
That's why the description goes with: "Requires https://github.com/lest/centos-rpm-builder-docker/pull/5"

File: Makefile [hunk 1]
```diff
@@ -68,9 +68,30 @@ all: auto manual
 manual: $(MANUAL)
 auto: $(AUTO_GENERATED)
 
+manual9: $(addprefix build9-,$(MANUAL))
 manual8: $(addprefix build8-,$(MANUAL))
 manual7: $(addprefix build7-,$(MANUAL))
 
+$(addprefix build9-,$(MANUAL)):
+	$(eval PACKAGE=$(subst build8-,,$@))
```
Comment (lest) [inline Makefile:76]:
```suggestion
	$(eval PACKAGE=$(subst build9-,,$@))
```

**Comment by karlism** [comment]
RedHat 9 has just been [released](https://www.redhat.com/en/blog/hot-presses-red-hat-enterprise-linux-9), I think it's best to wait until OL 9 is released and build packages with it.

**Comment by lfarkas** [comment]
almalinux 9 already released.

**Comment by karlism** [comment]
@lest, please merge this https://github.com/lest/centos-rpm-builder-docker/pull/8
I think it's best to build RHEL9 packages on Oracle Linux 9, similar to what we do now for RHEL8 packages.

**Comment by karlism** [comment]
@lest, I've switched package build to OL9, all looks good now. Can you please create packagecloud repositories and merge this?
I will address linting issues in separate PR

**Pull request merged**